<!-- {about} --><a name="about"></a>
  <p>(Imported via <a href="http://metazoa.ensembl.org">EnsemblMetazoa</a>)</p>
  <p>Within the butterflies, the combined traits of daytime flight and colour-vision have led to the evolution of complex colour patterns for both within- and between-species signalling. As long ago as 1862, Henry W. Bates, reflecting upon observations made during his stay in the Amazon basin, had shown that colour patterns for butterflies of the genus Heliconius were similar between species within any one area. Furthermore, the specific patterns of such Mullerian mimickry, whereby species share the cost of educating predators about their unpalatable nature, changes every 100-200 miles. Such characteristics provide exciting opportunities for genomic studies into the nature of adaptive variation, the origin of new species, and the genetic basis of complex traits. The Postman Butterfly, <em>Heliconius melpomene</em>, has large long wings with a red stripe down each forewing. It is poisonous, and has a geographical distribution from Central to South America. It is the first of the Heliconius butterflies to have its genome sequenced.</p>
  <p>Picture credit: Mauricio Linares, via <a href="http://www.heliconius.org/2006/h-m-melpomene/">Chris Jiggins</a> (University of Cambridge)</p>
<!-- {about} -->

<!-- {assembly} --><a name="assembly"></a>

  <h2 id="assembly" style="margin-bottom:5px">Assembly</h2>

  <p>From the abstract [1]: "The Heliconius butterflies are a widely studied adaptive radiation of 46 species spread across Central and South America, several of which are known to hybridise in the wild. Here, we present a substantially improved assembly of the 
Heliconius Melpomene genome, developed using novel methods that should be applicable to improving other genome assemblies produced using short read sequencing. Firstly, we whole genome sequenced a pedigree to produce a linkage map incorporating 99% of the 
genome. Secondly, we incorporated haplotype scaffolds extensively to produce a more complete haploid version of the draft genome. Thirdly, we incorporated ~20x coverage of Pacific Biosciences sequencing and scaffolded the haploid genome using an assembly of this 
long read sequence. These improvements result in a genome of 795 scaffolds, 275 Mb in length, with an N50 length of 2.1 Mb, an N50 number of 34 and with 99% of the genome placed and 84% anchored on chromosomes. We use the new genome assembly to confirm that the 
Heliconius genome underwent 10 chromosome fusions since the split with its sister genus Eueides, over a period of about 6 million years."</p>

<!-- {assembly} -->

<!-- {annotation} --><a name="annotation"></a>
  <h2 id="annotation" style="margin-bottom:5px">Annotation</h2>
  <p>From <a href="https://github.com/johnomics/Heliconius_melpomene_version_2">GitHub</a>, "The Hmel1 annotation was transferred to Hmel2 by merging Hmel1-2 and Hmel2 with HaploMerger to generate chain files, filtering the chain to retain only between-genome matches, using CrossMap to transfer features with the chains, and then using transfer_features.py to transfer as many features directly as possible, falling back on the CrossMap transfers where necessary."</p>
<!-- {annotation} -->
<!-- {references} -->
  <h2>References</h2>
  <ol>
    <li><a id="ref-01"></a><a href="http://www.ncbi.nlm.nih.gov/pubmed/26772750">Major Improvements to the Heliconius melpomene Genome Assembly Used to Confirm 10 Chromosome Fusion Events in 6 Million Years of Butterfly Evolution.
</a></li>
  </ol>
 <!-- {references} -->
